org.dpsoftware.FireflyLuciferin.appdata.xml

```diff
@@ -0,0 +1,74 @@
+            <li>Advanced algorithms for smooth colors transitions and color correction. Seeing is believing.</li>
+            <li>Wireless or cabled, local or remote? Choose your flavour with MQTT support and Home Assistant integration.</li>
+            <li>Multi monitor support with single or multiple instances.</li>
+            <li>Surround lighting with satellites.</li>
+            <li>Web browser firmware installer and Web Interface.</li>
+            <li>Programmable firmware, change your microcontroller's settings on the fly.</li>
+            <li>Frequent updates, upgrade your PC software and your firmware in one click.</li>
+            <li>Automatic switching between aspect ratios based on your video content.</li>
+            <li>Made from a gamer, for gamers. No added lag, stutter free.</li>
+            <li>Multi platform, Windows and Linux ready. macOS is coming when it's ready.</li>
+            <li>If you don't want to design your own PCB and you have a soldering iron, there is Luciferin's official PCB.</li>
+            <li>Have a question? Get answered on the community.</li>
+        </ul>
+    </description>
+    <screenshots>
+        <screenshot type="default">
+            <caption>Ambilight effect</caption>
+            <image type="source">https://raw.githubusercontent.com/sblantipodi/firefly_luciferin/dx12_pipeline/data/img/ambilight_show.png</image>
+        </screenshot>
+        <screenshot type="default">
+            <caption>Artist</caption>
+            <image type="Luciferin UI">https://raw.githubusercontent.com/sblantipodi/firefly_luciferin/master/data/img/settings_1.png</image>
+        </screenshot>
+        <screenshot>
+            <caption>Luciferin UI</caption>
+            <image type="source">https://raw.githubusercontent.com/sblantipodi/firefly_luciferin/master/data/img/settings_2.png</image>
+        </screenshot>
+        <screenshot>
+            <caption>Luciferin UI</caption>
+            <image type="source">https://raw.githubusercontent.com/sblantipodi/firefly_luciferin/master/data/img/settings_3.png</image>
+        </screenshot>
+        <screenshot>
+            <caption>Web Interface</caption>
+            <image type="source">https://raw.githubusercontent.com/sblantipodi/glow_worm_luciferin/master/assets/img/web_interface.jpg</image>
+        </screenshot>
+        <screenshot>
+            <caption>Home Assistant integration</caption>
+            <image type="source">https://raw.githubusercontent.com/sblantipodi/glow_worm_luciferin/master/assets/img/HA_mobile_client_screenshot.jpg</image>
+        </screenshot>
+        <screenshot>
+            <caption>Test Image</caption>
+            <image type="source">https://raw.githubusercontent.com/sblantipodi/firefly_luciferin/master/data/img/testimage.jpg</image>
+        </screenshot>
+    </screenshots>
+    <kudos>
+        <kudo>HiDpiIcon</kudo>
+    </kudos>
```

Comment:
```suggestion
```

this tag is no longer used for anything.

Reply:
fixed.